src/libs/ReportPreviewActionUtils.ts

```diff
@@ -102,6 +102,10 @@ function canApprove(report: Report, violations: OnyxCollection<TransactionViolat
         return false;
     }
 
+    if (!!transactions && transactions?.length > 0 && !transactions.some((transaction) => !isPending(transaction))) {
```

Comment:
```suggestion
    if (!!transactions && transactions?.length > 0 && transactions.every((transaction) => isPending(transaction))) {
```
Do you think to make it consistent how we have done for submit part?